SQL.js

```diff
@@ -44,6 +44,16 @@ class SqlStatement {
     return text.replace(/^\s+|\s+$/mg, '')
   }
 
+  get sql () {
+    let text = this.strings[0]
+
+    for (let i = 1; i < this.strings.length; i++) {
+      text += '?' + this.strings[i]
+    }
+
+    return text.replace(/^\s+|\s+$/mg, '')
```

Comment:
can you extract a function to generate these? both `sql` and ` text` are identical apart from the `'$'` and `'?'`?